src/DataService/Player/Player.php

```diff
@@ -0,0 +1,2460 @@
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $adPolicyId;
+
+    /**
+     * Indicates whether the player will feature sharing via email.
+     *
+     * @var bool
+     */
+    protected $allowEmail;
+
+    /**
+     * Indicates whether the player will provide an embed code for sharing on other sites.
+     *
+     * @var bool
+     */
+    protected $allowEmbed;
+
+    /**
+     * Indicates whether the player will be allowed to play video in fullscreen mode.
+     *
+     * @var bool
+     */
+    protected $allowFullScreen;
+
+    /**
+     * Indicates whether the player will make a shareable link available to users.
+     *
+     * @var bool
+     */
+    protected $allowGetLink;
+
+    /**
+     * Indicates whether the player will make an RSS link available to users.
+     *
+     * @var bool
+     */
+    protected $allowRss;
+
+    /**
+     * Indicates whether to include the search component.
+     *
+     * @var bool
+     */
+    protected $allowSearch;
+
+    /**
+     * Indicates whether the player should always display the play overlay.
+     *
+     * @var bool
+     */
+    protected $alwaysShowOverlay;
+
+    /**
+     * The value of the aspect ratio for the height of the media area.
+     *
+     * @var int
+     */
+    protected $aspectRatioHeight;
+
+    /**
+     * The value of the aspect ratio for the width of the media area.
+     *
+     * @var int
+     */
+    protected $aspectRatioWidth;
+
+    /**
+     * Indicates whether the player will start playback on load.
+     *
+     * @var bool
+     */
+    protected $autoPlay;
+
+    /**
+     * Indicates whether the player will self-initialize on load.
+     *
+     * @var bool
+     */
+    protected $autoInitialize;
+
+    /**
+     * URL for a custom background image.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $backgroundImageUrl;
+
+    /**
+     * Identifier for the color scheme assigned to this player.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $colorSchemeId;
+
+    /**
+     * The number of columns in the release list.
+     *
+     * @var int
+     */
+    protected $columns;
+
+    /**
+     * Reserved for future use. The default value is false.
+     *
+     * @var bool
+     */
+    protected $compatibilityMode;
+
+    /**
+     * XML layout content for the player control rack.
+     *
+     * @var string
+     */
+    protected $controlLayoutXml;
+
+    /**
+     * Height of the player control rack.
+     *
+     * @var int
+     */
+    protected $controlRackHeight;
+
+    /**
+     * Custom CSS content for the player.
+     *
+     * @var string
+     */
+    protected $customCss;
+
+    /**
+     * URLs to remote custom CSS content for the player.
+     *
+     * @var \Psr\Http\Message\UriInterface[]
+     */
+    protected $customCssUrls;
+
+    /**
+     * Custom HTML content for the player (not currently used).
```

Comment:
What does "not currently used" mean? Do we actually need this property?

Reply:
Another one that's what MPX documents.